README.md

```diff
@@ -6,168 +6,171 @@
-<img style="border:1px black solid;" height=165px src="READMEassets/dashboard.png">
-</div>
+Being inspired by the Cornucopia of various social hub this project has been developed taking into consideration about open source.
 
+Well, this is an Open Source Social networking hub which acts as a bridge between various Developers, Organisations and Open Source aspirants to elaborate on various things like Projects, Events, Discussion on various researches, Scholarships, Coding release and various other things updates.
 
-What is Donut?
---------
-Being inspired by the Cornucopia of various social hub this project has been developed taking into consideration about open source.
- 
- Well, this is an Open Source Social networking hub which acts as a bridge between various Developers, Organisations and Open Source aspirants to elaborate on various things like #Projects, #Events, #Discussion on various researches, #Scholarships, #Coding release and various other things updates. 
+The major priority of this project has been that this platform allows users to make their project "Open Sourced" and released them under various open source Organisations, experts which hold up a ring plate on this portal.
+
+This platform also makes users introduce and develops various solutions in the form of FOSS software to publish them for public use by integrating them with their social cause. Moreover, this project can be downloaded by any user, organization and can be used by them in their own custom way, making it run on their servers.
+
+### Frontend Technology Stack
+
+- CSS: Styling web pages, html files
+- Javascript: Primary programing language
+- ReactJS: Javascript library for building User Interfaces
+- Bootstrap: Styling framework.
+
+Learn more about the donut platform at the [official donut documentation](https://docs.codeuino.org/donut-social-networking-platform/).
+
+---
+
+# Setting up the work environment.
+
+Few things are required before setting up the project locally,
+
+- **A Web Browser** - You can install any browser as per your preference e.g. Internet Explorer,Chrome, FireFox, Safari, Opera etc. JavaScript works on any web browser on any OS.
+- **An Editor**- We prefer an editor which has built-in features of IntelliSense support and syntax error highlighter for speedy development. Download Visual Studio Code from [here](https://code.visualstudio.com/download).
+- **Node and Node Package Manager(NPM)**
+  Check if node and npm are already installed by running the following commands in the terminal
+  ` node -v npm -v `
+  If the node and npm are installed
+  If not, install it in,
+- **Windows & MacOS**
 
-The major priority of this project has been that this platform allows users to make their project "Open Sourced" and released them under various open source Organisations, experts which hold up a ring plate on this portal. 
+  Installing Node and NPM on Windows and macOS is straightforward because you can just use the provided installer:
```

Comment:
@ksraj123 I agree with you but I wanted to make sure that someone new would have no issue setting up the project and contributing to it since this was what @vaibhavdaren brought up during the last meeting.